compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -90,29 +90,20 @@ object Completion:
 
     val completionSymbolKind: Mode =
       path match
-        case untpd.Ident(_) :: untpd.Import(_, _) :: _ => Mode.ImportOrExport
-        case untpd.Ident(_) :: (_: untpd.ImportSelector) :: _ => Mode.ImportOrExport
-        case untpd.Literal(Constants.Constant(_: String)) :: _ => Mode.Term // literal completions
+        case untpd.Ident(_) :: (_: untpd.ImportSelector) :: _ => Mode.ImportOrExport // import scala.@@
```

Comment:
possibly also should be member?

Reply:
As far as I know member completions are fetched from symbol search. They are also source of extension methods outside the scope. ain this scenario we can only complete with direct members